tidy3d/plugins/adjoint/web.py

```diff
@@ -691,6 +694,9 @@ def run_local_bwd(
         callback_url=callback_url,
         verbose=verbose,
     )
+
+    sim_data_adj = sim_data_adj.normalize_adjoint_fields()
```

Comment:
To fix the backend tests, I had to add this to the server-side adjoint handling too. Basically any time you add something to the internals in `run_local`, we need to make sure it is also incorporated in the server-side run, so let me know.

I've fixed the backend tests in a commit on your backend branch.

Reply:
I see, thanks.